app/test_main.py

```diff
@@ -1,3 +1,26 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_human_age_if_animals_age_equal_zero() -> None:
+    animals = get_human_age(0, 0)
+    assert animals == [0, 0]
+
+
+def test_human_age_if_animals_age_less_than_first_human_age() -> None:
+    animals = get_human_age(7, 12)
+    assert animals == [0, 0]
+
+
+def test_human_age_if_animals_age_more_or_equal_than_first_human_age() -> None:
+    animals = get_human_age(15, 17)
+    assert animals == [1, 1]
```

Comment:
The expected result `[1, 1]` assumes that both animals have reached the first human age. Ensure that this aligns with the logic of `get_human_age`.